src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -72,7 +73,7 @@
     private final String authHeader;
 
     private final String baseApiPath;
-    
```

Comment:
I understand that you invoked a default formatter against the whole file - that's OK, but please split that formatting into a separate commit. It's hard to review the whole PR this way + in the history review it's easier to do "git blame" later to see when a line was changed and why.